src/components/curator-actions/child-bounties/ChildBountyCard.svelte

```diff
@@ -1,228 +1,142 @@
+	import ClaimChildBounty from './operations/ClaimChildBounty.svelte';
+	import CloseDownChildBounty from './operations/CloseDownChildBounty.svelte';
 	import CopyableAddress from '../../common/CopyableAddress.svelte';
-	import BatchChildBountyCalls from './operations/BatchChildBountyCalls.svelte';
-	import UnassignSubCurator from './operations/UnassignSubCurator.svelte';
 	import Currency from '../../Currency.svelte';
-	import { isCurator } from '../../../utils/isCurator';
+	import UnassignSubCurator from './operations/UnassignSubCurator.svelte';
 
 	export let childBounty: ChildBounty;
 	export let parentBounty: Bounty;
 
 	let assignSubCuratorOpen = false;
 	let unassignSubCuratorOpen = false;
-	let acceptSubCuratorRuleOpen = false;
+	let AcceptSubCuratorRoleOpen = false;
 	let closeDownChildBountyOpen = false;
 	let awardChildBountyOpen = false;
 	let claimChildBountyOpen = false;
 	let batchOpen = false;
-
-	let detailsExpended = false;
-	let closeDownExpended = false;
-
-	function handleMoreDetailsToggleClick() {
-		detailsExpended = !detailsExpended;
-	}
-
-	function initiateCloseDownToggleClick() {
-		closeDownExpended = !closeDownExpended;
-	}
 </script>
 
-<div class="childContainer bg-backgroundBounty">
+<div class="child-container">
 	<!-- Header Section -->
-	<div class="flex flex-col rounded-t-md pt-[16px] px-[11px] bg-backgroundBounty">
-		<span class="text-[18px] break-word">#{childBounty.id} {childBounty.description}</span>
-		<div class="flex justify-end bg-backgroundChildBountyDetails">
-			<span
-				class="status justify-end items-center text-xs flex-shrink-0 text-white rounded-md pt-1.5 px-3"
-				data-status={childBounty.status}
-			>
+	<div class="header">
+		<h2 class="title">#{childBounty.id} {childBounty.description}</h2>
+		<div class="status-container">
+			<span class="status" data-status={childBounty.status}>
 				{statusLabels[childBounty.status]}
 			</span>
 		</div>
 	</div>
 
 	<!-- Child Bounty Card Content -->
-	<div
-		class="flex flex-col bg-backgroundChildBountyDetails mt-2 mx-[10px] space-y-3 py-[12px] px-[6px]"
-	>
-		<div class="space-y-3">
-			<div class="flex justify-between">
-				<section>
-					<p class="text-xs">Value</p>
-					<p><Currency value={childBounty.value} /></p>
-				</section>
-				<section>
-					<p class="text-xs">Sub-curator Fee</p>
-					<p><Currency value={childBounty.fee} /></p>
-				</section>
-			</div>
-
-			<div class="flex-col space-y-2">
-				{#if childBounty.curator}
-					<section>
-						<p class="text-xs">Sub-Curator</p>
-						<CopyableAddress address={childBounty.curator || '-'} />
-					</section>
-				{/if}
-
-				{#if childBounty.beneficiary}
-					<section>
-						<p class="text-xs">Beneficiary</p>
-						<CopyableAddress address={childBounty.beneficiary} />
-					</section>
-				{/if}
+	<div class="card-content">
+		<section class="card-details">
+			<div class="elements">
+				<div>
+					<p class="text">Sub-Curator</p>
```

Comment:
I think we should choose between "Sub-Curator" and "Sub-curator" and use it consistently